tests/checks/integration/test_elastic.py

```diff
@@ -83,6 +85,8 @@
     "elasticsearch.merges.total.size": ("gauge", "indices.merges.total_size_in_bytes"),
     "elasticsearch.refresh.total": ("gauge", "indices.refresh.total"),
     "elasticsearch.refresh.total.time": ("gauge", "indices.refresh.total_time_in_millis", lambda v: float(v)/1000),
+    "elasticsearch.indices.translog.size_in_bytes": ("gauge", "indices.translog.size_in_bytes"),
+    "elasticsearch.indices.translog.operations": ("gauge", "indices.translog.operations"),
```

Comment:
I think the tests on `0.90.13` are failing because you're testing these 2 metrics on all the versions of elasticsearch. These metrics should only be listed in `ADDITIONAL_METRICS_POST_1_0_0 ` below.